crates/sovereign-sdk/rollup-interface/src/state_machine/soft_confirmation.rs

```diff
@@ -13,18 +13,19 @@ use serde::{Deserialize, Serialize};
 
 /// Contains raw transactions and information about the soft confirmation block
 #[derive(Debug, PartialEq, BorshSerialize)]
-pub struct UnsignedSoftConfirmation<'txs> {
+pub struct UnsignedSoftConfirmation<'txs, Tx> {
     l2_height: u64,
     da_slot_height: u64,
     da_slot_hash: [u8; 32],
     da_slot_txs_commitment: [u8; 32],
     txs: &'txs [Vec<u8>],
+    txs_new: &'txs [Tx],
```

Comment:
We can probably find a better name for this.

We can name `txs` as `serialized_txs` and leave `txs` as the unserialized ones.

Reply:
Or `txs` becomes `blobs` and `txs_new` becomes `txs`